lib/mobility/backends/active_record/key_value.rb

```diff
@@ -186,7 +186,12 @@ def visit_default(_)
           include const_set(module_name, callback_methods)
         end
 
-        include DestroyKeyValueTranslations
+        after_destroy do
+          klasses = [translations_class, *Mobility::Backends::ActiveRecord::KeyValue::Translation.descendants].uniq
```

Comment:
I reached a kind of compromise: we destroy any descendant of `Mobility::Backends::ActiveRecord::KeyValueTranslation`, plus `translations_class` if that class is defined and different. This may be more than is necessary but I think it's safest to ensure all translations are deleted.